### DISCUSSION TITLE
Text to Speech

### DISCUSSION THREAD

**Discussion by michalik-michal** [MacPaw/OpenAI#117] [Q&A] [answered]
Hi, will you be updating the package with lately released text to speech API?

**Comment by ingvarus-bc** [comment, marked as answer]
Hi, @michalik-michal 👋 
Yes, upcoming this week, implemented [here](https://github.com/MacPaw/OpenAI/pull/120)

**Comment by leventmolla** [comment]
Is there any timeline for the release of text-to-speech classes to be used with tts-1 and tts-1-hd? I see that the classes are there, but they haven't been worked into the latest 0.2.5 release.

**Reply by kalafus** [reply]
The endpoint is coded, works, and is in the demo as of 0.2.6. 🙏

**Reply by leventmolla** [reply]
Thanks, it works in this latest version.
